nlp/app.py

```diff
@@ -0,0 +1,35 @@
+from flask import Flask, request, jsonify
+from sklearn.externals import joblib
+from pathlib import Path
+
+app = Flask(__name__)
+clf = None
+tf = None
+
+@app.before_first_request
+def load():
+    global clf
+    global tf
+    modelFile = Path.cwd() / 'nlp/tweets_model.pkl'
+    vectFile = Path.cwd() / 'nlp/tweets_vectorizer.pkl'
+
+    clf = joblib.load(modelFile)
+    tf = joblib.load(vectFile)
+
+@app.route('/', methods=['POST'])
+def main():
+    global clf
+    global tf
```

Comment:
You already declared them as global above so you don't really need these here. You'd only need it if you were reassigning them a value or whateva.